front-end/package.json

```diff
@@ -8,12 +8,14 @@
 		"@testing-library/react": "^13.3.0",
 		"@testing-library/user-event": "^13.5.0",
 		"chart.js": "^3.6.0",
+		"chartjs-plugin-downsample": "^1.1.0",
```

Comment:
there is potential risk in this - only 312 weekly downloads and last updated 3 years ago. we need to be diligent in selecting packages to use for security tools, especially when they end up in the distributed package.